### ISSUE TITLE
Nothing Gets Triggered

### ISSUE THREAD

**Issue by depaolam** [dustindclark/homebridge-envisalink#64]
I'm sure I'm doing something dumb, but I've tried uninstalling and reinstalling the plugin multiple times and nothing gets triggered. When Homebridge launches, it starts out looking good, displaying the following:

[2/11/2021, 10:00:46 PM] [Envisalink] Initializing Envisalink platform...
[2/11/2021, 10:00:46 PM] [Envisalink] Configuring Envisalink platform,  Host: 192.168.1.107, port: undefined, type: DSC
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID 36f431e8-a869-4e88-b1aa-c18193e6f801 for accessory ID envisalink.1
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID 769cc457-047f-4c76-ab44-8fcd67570fcf for accessory ID envisalink.1.1
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID f8bb280b-c337-4136-a62a-12a2f83a81d8 for accessory ID envisalink.1.2
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID 8ce38bbc-fa2b-4e94-87b3-41f79280d7c6 for accessory ID envisalink.1.3
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID 1b5f4151-0309-4f1f-bf81-b9ca566253a9 for accessory ID envisalink.1.4
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID 7fc5db4e-7d90-4438-b09d-b94527f38d40 for accessory ID envisalink.1.5
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID e9e049f8-d4ae-403f-b127-89ab5c67b8ef for accessory ID envisalink.1.6
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID 253b190a-02ca-4510-ae7a-dafeb3a82aeb for accessory ID envisalink.1.7
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID e0b951f7-eb67-43e4-a55c-222c18b0bcf1 for accessory ID envisalink.1.8
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID 3a771336-4a29-4fc2-9b91-bbc0f172ae3e for accessory ID envisalink.1.9
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID fc23e1d5-9c6e-47d2-a21e-b5f78e3f8f3d for accessory ID envisalink.1.10
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID 4cb9ef68-cd3c-43cb-acd7-e93b800e993c for accessory ID envisalink.1.11
[2/11/2021, 10:00:46 PM] [Envisalink] Generated UUID a81b7292-17d9-4c56-8f1b-ff9526679aae for accessory ID envisalink.1.12
[2/11/2021, 10:00:46 PM] [Envisalink] Starting node alarm proxy...
[2/11/2021, 10:00:46 PM] [Envisalink] Zone Config: 12
[2/11/2021, 10:00:46 PM] [Envisalink] User Program Config: null
[2/11/2021, 10:00:46 PM] [Envisalink] Node alarm proxy started.  Listening for connections at: 0.0.0.0:4026
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Home Alarm'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Smoke Detectors'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'CO Detectors'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Septic Tank'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Motion 1'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Motion 2'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Motion 3'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Door 1'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Door 2'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Door 3'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Door 4'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Door 5'...
[2/11/2021, 10:00:46 PM] Initializing platform accessory 'Door 6'...

But opening doors, triggering motion detectors (haven't set off the smoke or CO alarms), nothing shows up in the Homebridge log, in the Home app, these devices never show as triggered. I have an Envisalink 3 and have made sure the IP address stays constant. I'm running Homebridge 1.1.7. If anyone has any thoughts or suggestions I'd very much appreciate it! Thanks in advance!

**Comment by Pblondin123** [comment]
are the motion sensors showing as 'open' in HB (full time)?   If so i neglected to add resistors on each zone which caused the zones to be all opened.   Added resistors and voila motion detectors got live.

**Comment by dustindclark** [comment]
Can you test this in 0.4.0-beta.3? Plugin has been rewritten, and I have re-verified motion detectors.

**Comment by Pblondin123** [comment]
Hi yes tested with success.  Motions come thru.  thanks

**Comment by dustindclark** [comment]
Great, thanks for verifying. Will get this published to live once I've figured out a way to verify smoke sensors.

**Comment by Pblondin123** [comment]
as a sidebar the new code reseted the devices in HK hence i lost 2 automations that were using those devices (when motion triggered then xyz).    not a biggy just stating in case some folks may have more automations.

**Comment by dustindclark** [comment]
Yeah, I noticed that as well. In fact, I noticed it when bumping the plugin version from 0.2.16 to 0.3.1-beta.0 (before the refactor). I think maybe Homebridge uses the plugin version as a part of the UUID generation logic, but I haven't stumbled across any documentation that confirms.

**Comment by depaolam** [comment]
Sorry, stupid noob question. Clearly there's a way to find the pre-release version and install it outside of the Homebridge UI. Anywhere you can point me so I might learn? Thanks!

**Comment by dustindclark** [comment]
You can do this in the Homebridge UI. Click Plugins, click the wrench next to homebridge-envisalink, then click "Install previous version". You should see the beta version listed there.

**Comment by depaolam** [comment]
Thank so much! I didn't realize I could do that. Unfortunately with v0.4.0-beta.3 I get a password error:

[11/23/2021, 2:12:23 PM] [Envisalink] Configuring Envisalink platform,  Host: 192.168.1.107 Port: 4025
[11/23/2021, 2:12:23 PM] [Envisalink] Node alarm proxy started.  Listening for connections at: 0.0.0.0:4026
[11/23/2021, 2:12:23 PM] [Envisalink] Panic buttons disabled. Will not install panic switches.
server bound
actual connected
Login Request Response: 3 
login requested... sending response...
sendcommand 005<***My Password***>6D
Command 005 Acknowledged
Login Request Response: 0 
Incorrect Password :(
actual disconnected

Version v0.2.18 works fine, but I noticed it seems to connect regardless of what I enter as a password. Nonetheless, I validated the password by logging in directly to the device.
